backends/platform/sdl/switch/switch.cpp

```diff
@@ -164,3 +166,29 @@ Common::HardwareInputSet *OSystem_Switch::getHardwareInputSet() {
 
 	return inputSet;
 }
+
+Common::String OSystem_Switch::getSystemLanguage() const {
+	u64 lang;
+	SetLanguage langcode;
+
+	setInitialize();
+	setGetSystemLanguage(&lang);
+	setMakeLanguage(lang, &langcode);
+
+	switch (langcode) {
+		case SetLanguage_JA:   return "ja_JP";
+		case SetLanguage_ENUS: return "en_US";
+		case SetLanguage_FR:   return "fr_FR";
+		case SetLanguage_FRCA: return "fr_FR";
+		case SetLanguage_DE:   return "de_DE";
+		case SetLanguage_IT:   return "it_IT";
+		case SetLanguage_ES:   return "es_ES";
+		case SetLanguage_ZHCN: return "zh_CN";
+		case SetLanguage_KO:   return "ko_KR";
+		case SetLanguage_NL:   return "nl_NL";
+		case SetLanguage_PT:   return "pt_PT";
+		case SetLanguage_RU:   return "ru_RU";
+		case SetLanguage_ZHTW: return "zh_HK";
```

Comment:
Why not zh_TW?

Reply:
yes, my mistake.

@rsn8887 can you fix that ?